products/ubuntu.md

```diff
@@ -116,6 +172,14 @@ releases:
     latest: "14.04.6"
     releaseDate: 2014-04-17
     latestReleaseDate: 2019-03-07
+-   releaseCycle: "12.04"
+    codename: "Precise Pangolin"
+    support: 2017-04-28
+    eol: 2017-04-28
+    extendedSupport: 2019-04-26
+    latest: "12.04"
+    releaseDate: 2012-04-26
+    latestReleaseDate: 2014-08-08
```

Comment:
Should be 2014-08-07

Reply:
Oh, looks like the distrowatch date is one day later than https://wiki.ubuntu.com/Releases. Nevermind, it will be changed by the next auto-update.